pandas/tests/indexing/test_iloc.py

```diff
@@ -1000,7 +1000,7 @@ def test_iloc_getitem_nonunique(self):
         ser = Series([0, 1, 2], index=[0, 1, 0])
         assert ser.iloc[2] == 2
 
-    def test_setitem_iloc_pure_position_based(self):
+    def test_iloc_setitem_pure_position_based(self):
```

Comment:
the comment about test naming applied to a few tests in the previous PR.

BTW i have a branch called "collect" in which i collect small edits that dont necessarily merit their own PRs, then periodically make a PR titled "assorted cleanups"

Reply:
Ah yeah you are right :)

When I started contributing someone told me to try avoiding cluttering things together from different prs, so I tried to split everything up as good as possible.

Will collect clean ups in the future :)

Reply:
> When I started contributing someone told me to try avoiding cluttering things together in one pr, which belong to different prs, so I tried to split everything up as good as possible.

In general this is good advice.  For the "assorted cleanups" PRs its important that the changes be pretty purely cosmetic so that they are super-easy to review.

> Will collect clean ups in the future :)

It works for me but not for everyone.  You find what works for you.

Reply:
Thanks for the help. Will try to collect them in the future if this is doable